{"timestamp":"2025-08-21T09:37:52.463604","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-08-21T09:37:52.464359","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/fetchin_cleaning.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-08-21T09:37:52.548810Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-21T09:37:52.549361Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-21T09:37:52.549959Z","level":"info","event":"Current task name:clean_data","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-21T09:37:52.550263Z","level":"info","event":"Dag name:weather_to_snowflake_pipeline","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-21T09:37:53.701335Z","level":"error","event":"Traceback (most recent call last):","chan":"stderr","logger":"task"}
{"timestamp":"2025-08-21T09:37:53.701827Z","level":"error","event":"  File \"/opt/airflow/dags/cleaning_data.py\", line 177, in <module>","chan":"stderr","logger":"task"}
{"timestamp":"2025-08-21T09:37:53.704602Z","level":"error","event":"    data_cleaning()","chan":"stderr","logger":"task"}
{"timestamp":"2025-08-21T09:37:53.705022Z","level":"error","event":"  File \"/opt/airflow/dags/cleaning_data.py\", line 73, in data_cleaning","chan":"stderr","logger":"task"}
{"timestamp":"2025-08-21T09:37:53.706954Z","level":"error","event":"    date_times = pd.to_datetime(date_times.dropna().unique()).dt.floor('H')","chan":"stderr","logger":"task"}
{"timestamp":"2025-08-21T09:37:53.707432Z","level":"error","event":"                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^","chan":"stderr","logger":"task"}
{"timestamp":"2025-08-21T09:37:53.707762Z","level":"error","event":"AttributeError: 'DatetimeArray' object has no attribute 'dt'","chan":"stderr","logger":"task"}
{"timestamp":"2025-08-21T09:37:53.871162","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"CalledProcessError","exc_value":"Command '['python', 'dags/cleaning_data.py']' returned non-zero exit status 1.","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":890,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1177,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":217,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":240,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/fetchin_cleaning.py","lineno":16,"name":"clean_data"},{"filename":"/usr/local/lib/python3.12/subprocess.py","lineno":571,"name":"run"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-08-21T09:37:53.873101Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-21T09:37:53.873533Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-21T09:37:53.873965Z","level":"info","event":"Task:<Task(PythonOperator): clean_data>","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-21T09:37:53.874317Z","level":"info","event":"Failure caused by Command '['python', 'dags/cleaning_data.py']' returned non-zero exit status 1.","chan":"stdout","logger":"task"}
